keyring.go

```diff
@@ -32,20 +39,30 @@ func Open(name string, backend string) (Keyring, error) {
 	return op(name)
 }
 
+// Item is an, uh, item that is stored on the keyring
 type Item struct {
 	Key         string
 	Data        []byte
 	Label       string
 	Description string
-	TrustSelf   bool
+	// macOS only, set false if you want the password prompt to appear every time
```

Comment:
I reckon the API right now isn't quite right, but probably beyond current PR scope to fix. I reckon it needs a bit more thought, I'm not quite sure what the best approach is to support functionality specific to a backend while presenting a generic interface as your API.